pkg/controllers/resourceapply/resourceapply.go

```diff
@@ -129,13 +129,17 @@ func applyConfigMap(ctx context.Context, client coreclientv1.Client, recorder re
 
 func applyDeployment(ctx context.Context, client coreclientv1.Client, recorder record.EventRecorder, requiredOriginal *appsv1.Deployment) (bool, error) {
 	required := requiredOriginal.DeepCopy()
-	err := SetSpecHashAnnotation(&required.ObjectMeta, required.Spec)
-	if err != nil {
+	if err := annotatePodSpecWithRelatedConfigsHash(ctx, client, required.Namespace, &required.Spec.Template); err != nil {
+		klog.V(5).Infof("Can not check related configs and secrets: %w", err)
```

Comment:
i looked through the code and we seem to have logs at levels `1`, `2`, `3`, and `4`. i wonder if we should maybe use 3 or 4 for these?

imo, 3 looks like a good target for our failure logs (see pkg/cloud/vsphere/vsphere_cloud_config/config.go)

Reply:
I updated loglevel, and also added info about the resource itself there.

https://github.com/openshift/cluster-cloud-controller-manager-operator/pull/227/files#diff-8a4ec169e12f51d19bc1aae0acb142b300939b46e8337ee467621b8f09eff6c6R148